src/openrct2/SpriteIds.h

```diff
@@ -1777,15 +1637,158 @@ enum : ImageIndex
 
     SPR_G2_END,
 
-    SPR_CSG_BEGIN = SPR_G2_END,
+    SPR_FONT_BEGIN = SPR_G2_END,
+    // Sprite font characters
```

Comment:
Considering the limited scope of these ids, and given this always starts numbering at `SPR_G2_END`, perhaps this is a good opportunity to move these ids to their own header as well? I'm thinking the drawing namespace/folder would be good. We could do the same with other groups when we split them off to their own files.

Reply:
That won't work, as the CSG sprite IDs are dependent on these.